jekyll/_cci2/orb-author.md

```diff
@@ -9,279 +9,289 @@ version:
 - Cloud
 ---
 
+* TOC
+{:toc}
+
 ## Introduction
 
-Once you have configured your platform to work with CircleCI orbs, you are ready to begin authoring your own orb. CircleCI has attempted to make this process simple and easy, eliminating any potential confusion by enabling you to use inline templates and other pre-configured templates to quickly author a new orb. The sections below describe the process of authoring your own orb.
+This orb authoring guide assumes you have read the [Introduction to authoring an orb]({{site.baseurl}}/2.0/orb-intro) document and claimed your namespace. At this point you are ready to develop and orb.
+
+Whether you are writing your first orb or getting ready for production level, we recommend using our [orb development kit](#orb-development-kit) to get started. Alternatively, as orbs are packages of [reusable configuration]({{site.baseurl}}/2.0/reusing-config), they can be written [manually]({{site.baseurl}}/2.0/orb-author-validate-publish), as singular `yaml` files, and published using our [circleci orb cli]({{site.baseurl}}/2.0/local-cli/#installation).
 
-## Orbs Quickstart
+## Orb Development Kit
 
-Before you create your first orb, please see the following notes:
+The Orb Development Kit refers to a suite of tools that work together to simplify the orb development process, with automatic testing and deployment on CircleCI.
```

Comment:
I wonder if it would be good to mention here that the tools are integrated into the CircleCI CLI?

Reply:
So I was thinking of doing something like this but I wasnt exactly sure how to present it.

I would say the orb dev kit is made up of:
  - Orb Project Template
  - Orb Pack
  - Orb Init
  - Orb Tools Orb

Orb init being the "main" piece the user interacts with. @rosieyohannan any thoughts?

Reply:
Yeah that would be a useful summary - could we add this list with a little description of each and a link to further info/readme/CLI help?

Reply:
@KyleTryon I'm going to add this list in to the doc